README.md

```diff
@@ -393,24 +393,70 @@ get removed from the DOM, and if so should there be cases where it remains?
 
 The standard toast will come with these default styles,
 which developers will be able to change to customize look and feel.
+They are similar to [the default styles for `<dialog>`](https://html.spec.whatwg.org/multipage/rendering.html#flow-content-3),
+which are very minimal.
+
+```css
+std-toast {
+    position: fixed;
+    offset-block-end: 0;
+    offset-inline-end: 0;
+    border: solid;
+    padding: 1em;
```

Comment:
Maybe add `margin: 1em` here? Every library in the study group had their default toast slightly offset from the edge of the screen.

Reply:
I was unsure. I'll change the position to add a 1em offset.